telegram/forcereply.py

```diff
@@ -31,31 +31,48 @@ class ForceReply(ReplyMarkup):
     to sacrifice privacy mode.
 
     Objects of this class are comparable in terms of equality. Two objects of this class are
-    considered equal, if their :attr:`selective` is equal.
+    considered equal, if their :attr:`selective`, and :attr:`input_field_placeholder` are equal.
 
     Args:
         selective (:obj:`bool`, optional): Use this parameter if you want to force reply from
             specific users only. Targets:
 
-            1) Users that are @mentioned in the text of the Message object.
-            2) If the bot's message is a reply (has reply_to_message_id), sender of the
-               original message.
+            1) Users that are @mentioned in the :attr:`~telegram.Message.text` of the
```

Comment:
```suggestion
            1) Users that are @mentioned in the :attr:`telegram.Message.text` of the
```

Reply:
that's actually intended - it adds the link to the doc and shows as '[text](https://python-telegram-bot.readthedocs.io/en/stable/telegram.message.html#telegram.Message.text)' instead of '[telegram.Message.text](https://python-telegram-bot.readthedocs.io/en/stable/telegram.message.html#telegram.Message.text)', so the resulting sentence makes little more sense

Reply:
ah, good to know, thought it was a typo. never mind, then :)